.github/workflows/deploy-to-eks.yaml

```
@@ -2,21 +2,29 @@ name: Create EKS cluster, deploy CKF and run bundle test
 on:
   workflow_dispatch: # This event allows manual triggering from the Github UI
     inputs:
+      bundle_version:
+        description: 'Comma-separated list of bundle versions e.g. "1.7","1.8"'
+        default: '"1.7","1.8","latest"'
+        required: true
+      k8s_version:
+        description: 'Kubernetes version to be used for the AKS cluster'
+        required: false
       uats_branch:
         description: 'Branch to run the UATs from e.g. main or track/1.8. By default, the workflow uses main.'
         required: false
-    secrets:
-      BUNDLE_KUBEFLOW_EKS_AWS_ACCESS_KEY_ID:
-        required: true
-      BUNDLE_KUBEFLOW_EKS_AWS_SECRET_ACCESS_KEY:
-        required: true
   schedule:
     - cron: "23 0 * * 2"
 jobs:
   deploy-ckf-to-eks:
     runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        bundle_version: ${{ fromJSON(format('[{0}]', inputs.bundle_version || '"1.7","1.8","latest"')) }}
```

Do we still need to specify the versions here? I believe this will take either the defaults or the inputs given in `bundle_version`, right?

```suggestion
        bundle_version: ${{ fromJSON(format('[{0}]', inputs.bundle_version )) }}
```

---

Oh, gotcha!